source/Activated.js

```diff
@@ -35,6 +35,13 @@ const Activated = ({
     setIsUnloading(true)
   }, [])
 
+  const handleEscClick = useCallback(e => {
```

Comment:
As you don't really "click" escape, we should probably name this `handleKeydown`, as `keydown` is the event we're listening for

Reply:
Applied!